constant.py

```diff
@@ -0,0 +1,3 @@
+class Status:
```

Comment:
can you give context class, EventStatus, or something meaningful instead of status?